build.sbt

```diff
@@ -36,17 +36,17 @@ lazy val library =
       "com.fasterxml.jackson.core"       % "jackson-databind"        % Version.jackson % Compile,
       "com.fasterxml.jackson.core"       % "jackson-core"            % Version.jackson % Compile,
       "com.fasterxml.jackson.core"       % "jackson-annotations"     % Version.jackson % Compile,
-      "com.fasterxml.uuid"               % "java-uuid-generator"     % "3.1.5"         % Compile
+      "com.fasterxml.uuid"               % "java-uuid-generator"     % "3.2.0"         % Compile
```

Comment:
Any reason for it to be excluded from the Version object?

Reply:
Just bumped the version. But I agree with your sentiment and added it to the Version object.